android/src/main/kotlin/com/onegini/mobile/sdk/flutter/models/CustomRegistrationModel.kt

```diff
@@ -0,0 +1,6 @@
+package com.onegini.mobile.sdk.flutter.models
+
+import androidx.annotation.Keep
+
+@Keep
```

Comment:
Usually we do not add annotations in the classes but add those into `proguard-rules.pro` files. I'm not sure about flutter though, if this is the proper way to do that then I'm fine. If you can add these into similar `.pro` file then that would be recommended approach. See https://github.com/onewelcome/example-app-android/blob/b4240fc95587cde8daef7be61b7002ffe9838426/app/proguard-rules-example-app.pro

Reply:
removed the keep lines; as there was already a proguard file that covered these model files